examples/proximity/main.py

```diff
@@ -37,7 +37,7 @@
     Label:
         text: 'Does Proximity Sensor detect something?'
     Label:
-        text: 'Yes' if root.is_near else 'No'
+        text: root.is_near
```

Comment:
@tshirtman I was trying to remove this condition statement from here. That's why String Property.

Reply:
I don't think it's a good idea to change the api to simplify a particular example, especially since a lot of other usages would change from:

    if obj.is_near:
        do_something()

to

    if obj.is_near == 'True':
         do_something()

which is really un-pythonic, the api is better when a boolean value is represented as a boolean.

Reply:
Also, you could just change the example to this, to achieve the same effect as your previous solution.

    text: str(root.is_near)

Reply:
"text: str(root.is_near)"
👍